src/components/modal.js

```diff
@@ -0,0 +1,24 @@
+import A11yDialog from "a11y-dialog";
+
+/**
+ *@param {string } modalClass
+ * @param {string} contentId
+ * @param {function} onShow
+ */
+const init = (modalClass, contentId, onShow = () => {}) => {
+
+  const modalClasses = document.querySelectorAll(`[class*=${modalClass}]`);
+
+  modalClasses.forEach(e => {
+    const dialog = document.getElementById(e.id);
```

Comment:
I think this line could possible be changed just to 

```js
const dialog = modelElm;
```

but not 100% sure.

Reply:
done